csrc/scheduler/pointwise.cpp

```diff
@@ -296,9 +296,7 @@ std::unique_ptr<PointwiseParams> getPointwiseHeuristics(
 
   int64_t max_dtype_size_bit_for_vectorization = 0;
   // ugly WAR.
```

Comment:
Can you please remove this comment and put a proper comment at a proper location?